client/extensions/woocommerce/components/table/table-row/index.js

```diff
@@ -3,14 +3,38 @@
  */
 import React, { PropTypes } from 'react';
 import classnames from 'classnames';
+import page from 'page';
 
-const TableRow = ( { className, isHeader, children, ...props } ) => {
+/**
+ * Internal dependencies
+ */
+import onKeyDownCallback from 'woocommerce/lib/keydown-callback';
+
+const TableRow = ( { className, isHeader, href, children, ...props } ) => {
 	const rowClasses = classnames( 'table-row', className, {
 		'is-header': isHeader,
 	} );
 
+	if ( ! href ) {
+		return (
+			<tr className={ rowClasses } { ...props }>
+				{ children }
+			</tr>
+		);
+	}
+
+	const goToHref = () => {
+		page( href );
+	};
+
 	return (
-		<tr className={ rowClasses } { ...props }>
+		<tr
+			className={ rowClasses + ' has-action' }
+			role="button"
+			tabIndex="0"
+			onClick={ goToHref }
+			onKeyDown={ onKeyDownCallback( goToHref ) }
```

Comment:
I tried to test this, but am getting errors on the `products` page. I'm wondering what role focus plays in this regard.